roles/cifmw_cephadm/tasks/pre.yml

```diff
@@ -158,3 +158,49 @@
     mode: "0644"
     config_overrides: "{{ cifmw_cephadm_conf_overrides }}"
     config_type: ini
+
+- name: Check the list of services
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+  shell: |
+    oc rsh openstackclient openstack service list
+    oc rsh openstackclient openstack user list
+    oc rsh openstackclient openstack role list
+    oc rsh openstackclient openstack endpoint list
+    oc get pods
+  register: service_list
+  delegate_to: localhost
+
+- name: print service list
+  debug:
+    msg: "{{ service_list }}"
+
+- name: Check if swift endpoint is already created
```

Comment:
We probably don't need this task since checking if swift is enabled or disabled should be sufficient.